src/richText.js

```diff
@@ -49,6 +49,8 @@ define([
     var CASE_REF_REGEX = /^\`?#case\//,
         FORM_REF_REGEX = /^\`?#form\//,
         REF_REGEX = /^\`?#(form|case)\//,
+        // not sure where this comes from... just grabbed from inspect element
+        CKEDITOR_WHITESPACE_CONSTANT = '​​​​​​​',
```

Comment:
This is 7 zero width spaces (http://www.fileformat.info/info/unicode/char/200B/index.htm). Can't use less or more or they show up in the xml. Guessing it's a separator in ckeditor, but I couldn't find it in their code.